core/modules/system/css/token.css

```diff
@@ -1,59 +1,171 @@
-/**
- * Styles for displaying the token tree and dialog.
- */
-/* Normally we would use a mobile-first media query, but in this case we only
-   apply max-width on smaller screens. Max-width/max-height settings interfere
-   with jQuery UI dialog resizing, but on a small screen we'll just take up
-   all available room. */
-.token-dialog {
-  max-width: none;
-  max-height: none;
-}
-@media only screen and (max-width: 600px) {
-  .token-dialog {
-    width: 100% !important;
-    max-width: 100%;
-    max-height: 100%;
-  }
-}
-
-.token-tree {
-  font-size: 0.85em;
-}
-.token-tree td {
+@charset 'UTF-8';
```

Comment:
It looks like browsers assume "UTF-8" by default (https://developer.mozilla.org/en-US/docs/Web/CSS/@charset). Is this necessary?

Reply:
Done